stable/datadog/values.yaml

```diff
@@ -1,290 +1,256 @@
+  ## It is possible to specify docker registry credentials
+  ## See https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+  #
+  #  pullSecrets:
+  #    - name: "<REG_SECRET>"
 
 datadog:
-  ## You'll need to set this to your Datadog API key before the agent will run.
+
+  ## @param apiKey - string - required
+  ## Set this to your Datadog API key before the Agent runs.
   ## ref: https://app.datadoghq.com/account/settings#agent/kubernetes
-  ##
-  # apiKey:
+  #
+  #  apiKey: <DATADOG_API_KEY>
 
+  ## @param securityContext - object - optional
   ## You can modify the security context used to run the containers by
   ## modifying the label type below:
-  # securityContext:
-  #   seLinuxOptions:
-  #     seLinuxLabel: "spc_t"
+  #
+  #  securityContext:
+  #    seLinuxOptions:
+  #      seLinuxLabel: "spc_t"
 
+  ## @param apiKeyExistingSecret - string - optional
   ## Use existing Secret which stores API key instead of creating a new one
-  # apiKeyExistingSecret:
+  #
+  #  apiKeyExistingSecret: <DATADOG_API_KEY_SECRET>
 
-  ## If you are using clusterAgent.metricsProvider.enabled = true, you'll need
-  ## a datadog app key for read access to the metrics
-  # appKey:
+  ## @param appKey - string - optional
+  ## If you are using clusterAgent.metricsProvider.enabled = true, you must set
+  ## a Datadog application key for read access to your metrics.
+  #
+  #  appKey: <DATADOG_APP_KEY>
 
+  ## @param appKeyExistingSecret - string - optional
   ## Use existing Secret which stores APP key instead of creating a new one
-  # appKeyExistingSecret:
+  #
+  #  appKeyExistingSecret: <DATADOG_APP_KEY_SECRET>
 
+  ## @param name - string - required
   ## Daemonset/Deployment container name
   ## See clusterAgent.containerName if clusterAgent.enabled = true
-  ##
+  #
   name: datadog
 
-  # The site of the Datadog intake to send Agent data to.
-  # Defaults to 'datadoghq.com', set to 'datadoghq.eu' to send data to the EU site.
-  # site: datadoghq.com
+  ## @param site - string - optional - default: 'datadoghq.com'
+  ## The site of the Datadog intake to send Agent data to.
+  ## Set to 'datadoghq.eu' to send data to the EU site.
+  #
+  #  site: datadoghq.com
 
-  # The host of the Datadog intake server to send Agent data to, only set this option
-  # if you need the Agent to send data to a custom URL.
-  # Overrides the site setting defined in "site".
-  # dd_url: https://app.datadoghq.com
+  ## @param dd_url - string - optional - default: 'https://app.datadoghq.com'
+  ## The host of the Datadog intake server to send Agent data to, only set this option
+  ## if you need the Agent to send data to a custom URL.
+  ## Overrides the site setting defined in "site".
+  #
+  #  dd_url: https://app.datadoghq.com
 
+  ## @param logLevel - string - required
   ## Set logging verbosity.
   ## ref: https://github.com/DataDog/docker-dd-agent#environment-variables
```

Comment:
this links to agent5
We can probably remove the "Note: For Agent 6" line and just state the valid log levels, as the chart only supports A6 now